README.md

```diff
@@ -2,26 +2,35 @@
   <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_docs" target="_blank" rel="noopener noreferrer">
     <picture>
       <source media="(prefers-color-scheme: dark)" srcset="https://images.clerk.dev/static/logo-dark-mode-400x400.png">
-      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="64">
+      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="120">
     </picture>
   </a>
   <br />
 </p>
 
-# Clerk Documentation
+<h1 align="center">
+  Clerk Documentation
+</h1>
 
-This repository contains the content for [Clerk's documentation](https://clerk.com/docs).
+Clerk is more than authentication, and we want to convey every feature of Clerk's as best as we can to make authentication and user management simpler for everyone. Explaining these concepts can sometimes be tricky, but with the help of a community, we can collaborate, share ideas, find the right words, and achieve greater clarity.
 
-### I've found an issue with your documentation
+We encourage you to contribute to Clerk's documentation! You can get started by following [this guide](./CONTRIBUTING.md).
 
-If you have found an issue with our documentation, please file an issue or create a pull request to fix it.
+> This repository contains the open-source documentation *content* for [Clerk docs](https://clerk.com/docs).  The *source code* for the docs site is not yet open source, so if you have any questions or suggestions about *non-content* related matters, please [reach out](https://clerk.com/support).
 
-### I have feedback
+## Found an issue?
 
-Feel free to create an issue with the feedback label. We will take the feedback and iterate on it.
+If you have found an issue with our documentation, please create an [issue](https://github.com/clerkinc/clerk-docs/issues).
 
-### How do I contribute?
+If it's a quick fix, such as a misspelled word or a broken link, feel free to skip creating an issue.  
+Go ahead and create a [pull request](https://github.com/clerkinc/clerk-docs/pulls) with the solution :rocket:
 
-If you've ever wanted to contribute to open source and want to help improve Clerk's documentation, now is your chance!
+## Want to leave feedback?
 
-See the [contributing](./CONTRIBUTING.md) document for more information on how to successfully contribute to Clerk's documentation.
+Feel free to create an [issue](https://github.com/clerkinc/clerk-docs/issues) with the **feedback** label. Our team will take a look at it and get back to you as soon as we can!
+
+## Connect with us
+
+You can discuss ideas, ask questions, and meet others from the community in our [Discord](https://discord.com/invite/b5rXHjAg7A). 
+
+If you prefer, you can also find support through our [Twitter](https://twitter.com/ClerkDev), or you can [email](mailto:support@clerk.dev) us!
```

Comment:
We might want to check with @perkinsjr to see if encouraging people to use Twitter for support is something we want to recommend. 💭

Reply:
that's actually straight from our /support page

Reply:
Ah, nice! That's validation enough for me then.